crypto/dh/dh_pmeth.c

```diff
@@ -455,6 +455,7 @@ static int pkey_dh_derive(EVP_PKEY_CTX *ctx, unsigned char *key,
     }
 #ifndef OPENSSL_NO_CMS
     else if (dctx->kdf_type == EVP_PKEY_DH_KDF_X9_42) {
+
```

Comment:
This empty line seem misplaced, I suppose you wanted it after `size_t Zlen = 0;`

Reply:
I missed them at my review after git rebase. I was concentrated on the change of returned value in pkey_dh_derive